lib/interpreters/periodic_visits.rb

```diff
@@ -212,6 +216,9 @@ def build_vehicle(vrp, vehicle, vehicle_day_index, rests_durations)
 
     def generate_vehicles(vrp)
       rests_durations = Array.new(vrp.vehicles.size, 0)
+
+      # Repopulate Models::Unit
+      vrp.units.each{ |unit| Models::Unit.insert(unit) } if Models::Unit.all.empty?
```

Comment:
Ok ! The problem comes from the tests and not the code base ;)

Reply:
Ok, the main problem comes from the fact that load_vrps load multiple vrps, but make a delete_all before loading any new vrp. So whenever there is actually multiple vrp to load, the objects refered into the firsts are erased

Reply:
Ah :face_exhaling:  yeah, `load_vrps` unfortunately does something that cannot be done via API, can't we 
1. move the `load_vrps` to a `yield/block` type of logic so that, we create the model objects just before the test  (instead of creating all before hand within load_vrps)
2. or at least, do the model::object.inser on the test size via a test_helper function that takes the vrp object and "populates" the model database?
What do you say, easy/hard?